snapcraft.yaml

```diff
@@ -1602,6 +1612,7 @@ parts:
     organize:
       commands/snap-query: bin/
       hooks/: snap/hooks/
+      wrappers/lxd-stophook: bin/lxd
```

Comment:
I missed that change in my previous review. I much prefer it to the previous extra part so thanks!